docs/reference/azure/apis.mdx

```diff
@@ -0,0 +1,25 @@
+Nitric APIs are deployed to Azure using [Azure API Management](https://azure.microsoft.com/en-us/services/api-management/).
+
+## Azure Resources
+
+The following resources are created when deploying APIs to Azure:
+
+- API Management Service
+- App Service Functions
```

Comment:
We're using Container Apps now (they're currently in preview so we'll need to add that caveat), if we're clever we'll eventually write plugins that hook into their Dapr operator to do our other stuff as well i.e. PubSub.